### PR TITLE
New Sounds

### PR DESCRIPTION
**Pull request by WWWWW-Git**
## About The Pull Request

Adds new sounds, music and a new title screen for the lobby.

## Why It's Good For The Game

The new title screen should thematically make the server seem more in line with what it is meant to be. And the new sounds and music should also help adding immersion to the game.

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Comment by NPC1314** [comment]
Overwrites a number of sounds and music files, to find out if the sounds are even acceptable I'll have to download the branch and check each file =/

**Comment by NPC1314** [comment]
Testmerge to check out the new audio tracks and title.

**Comment by bowlofcereal** [comment]
The doors sound really bad. Same for doorknocking.

File: code/game/objects/structures/mineral_doors.dm [hunk 1]
```diff
@@ -206,7 +206,7 @@
 				return
 		if(world.time >= last_bump+20)
 			last_bump = world.time
-			playsound(src, 'sound/foley/doors/knocking.ogg', 100)
+			playsound(src, 'sound/foley/doors/knocking2.ogg', 100)
```
Comment (NPC1314) [inline code/game/objects/structures/mineral_doors.dm:209]: revert to old knocking, we will use the new knocking too but in another way

File: code/game/objects/structures/mineral_doors.dm [hunk 2]
```diff
@@ -629,7 +629,7 @@
 	name = "door"
 	desc = ""
 	icon_state = "woodhandle"
-	openSound = 'sound/foley/doors/creak.ogg'
+	openSound = list('sound/foley/doors/creak.ogg', 'sound/foley/doors/creak2.ogg')
```
Comment (NPC1314) [inline code/game/objects/structures/mineral_doors.dm:632]: creak2 do be too anemic, drop it for now, could be used for opening some wooden case or such but not the door

**Comment by Optimism333** [comment]
The new knocking and door opening sounds aren't an upgrade, in my opinion.

Not sure how I feel about this track, either:

https://github.com/user-attachments/assets/3901bff2-ea02-4b11-b599-0df1ca06d9ef

I like this one a bit better:

https://github.com/user-attachments/assets/111a49ba-d980-41bf-8b1a-dd5d94d9d1dc

**Comment by NPC1314** [comment]
Door creak and knock should be the old since that commit yesterday, havent had chance to test live tho.

The old tracks should be preserved so could possibly be added as alt tracks in a later pass, but besides that, things looks like its good for a merge soon.

**Review by NPC1314 — APPROVED** [review]

**Review by NPC1314 — APPROVED** [review]

**Review by NPC1314 — APPROVED** [review]

**Pull request merged**